### PR TITLE
workload: merge master into workload and bump packet version

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: libraries/networking/src/udt/PacketHeaders.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ PacketVersion versionForPacketType(PacketType packetType) {
         case PacketType::EntityEdit:
         case PacketType::EntityData:
         case PacketType::EntityPhysics:
-            return static_cast<PacketVersion>(EntityVersion::CollisionMask16Bytes);
+            return static_cast<PacketVersion>(EntityVersion::YieldSimulationOwnership);
```
Comment (AndrewMeadows) [inline libraries/networking/src/udt/PacketHeaders.cpp]: The "yield simulation ownership" behavior is new and does not play well with interfaces who do not follow the same logic.  Therefore I'm bumping the packet version for talking to the entity-server.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10353/HighFidelity-Beta-PR13299-d220d43.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10353/HighFidelity-Beta-PR13299-d220d43.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10353/HighFidelity-Beta-Interface-PR13299-d220d43.dmg

**Pull request merged**